react/Checkbox/Checkbox.less

```diff
@@ -40,16 +40,16 @@
     .focus();
   }
 
-  .input:checked + .label & {
-    background-color: @sk-mid-gray-light;
+  .label & {
+    background-color: @sk-white;
   }
 }
 
 .checkMark {
   display: block;
   width: 100%;
   height: 100%;
-  fill: @sk-black;
+  fill: @sk-focus;
```

Comment:
This isn't really semantically correct, because if we tweak our focus styles, it would be unexpected to *also* be changing the colour of a tick. We might need to introduce another variable here, like `@sk-form-accent`?